docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -124,23 +124,22 @@ of domestic (:ref:`8 <census_prl_domestic_migration>`) and international (:ref:`
 by household and individual simulants, as well as changes to geographic
 location and household id.
 
-On top of this, we will layer attributes relevant to PRL: mailing
-addresses for each household (11); first, middle, and last names for
-each simulant (12); date of birth (13); intended-to-be-unique
+On top of this, we will layer attributes relevant to PRL: residential (11)
+and mailing addresses for each household (12); first, middle, and last names for
+each simulant (13); date of birth (14); intended-to-be-unique
 identification number modeling SSN that is missing for some and not
-actually unique for others (14); and periodic survey, census, and registry
-observations with realistic noise (15).
+actually unique for others (15); and periodic survey, census, and registry
```

Comment:
this might be in an update that i missed -- but are you guys still intending this? that SSNs are not unique for some sims? or is this referring only to the outputs in a census or set of tax forms?